[2023-07-28 22:13:24,419] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_etl_paises.crear_tabla_ scheduled__2023-07-09T00:00:00+00:00 [queued]>
[2023-07-28 22:13:24,830] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_etl_paises.crear_tabla_ scheduled__2023-07-09T00:00:00+00:00 [queued]>
[2023-07-28 22:13:24,844] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-07-28 22:13:24,849] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2023-07-28 22:13:24,856] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-07-28 22:13:25,400] {taskinstance.py:1397} INFO - Executing <Task(PostgresOperator): crear_tabla_> on 2023-07-09 00:00:00+00:00
[2023-07-28 22:13:25,523] {standard_task_runner.py:52} INFO - Started process 957 to run task
[2023-07-28 22:13:26,285] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_etl_paises', 'crear_tabla_', 'scheduled__2023-07-09T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/dag_etl_paises.py', '--cfg-path', '/tmp/tmppvbfsvgy', '--error-file', '/tmp/tmp8b3mmpni']
[2023-07-28 22:13:26,431] {standard_task_runner.py:80} INFO - Job 3: Subtask crear_tabla_
[2023-07-28 22:13:32,201] {task_command.py:371} INFO - Running <TaskInstance: dag_etl_paises.crear_tabla_ scheduled__2023-07-09T00:00:00+00:00 [running]> on host 42f90387d6b2
[2023-07-28 22:13:35,316] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Belenpintor
AIRFLOW_CTX_DAG_ID=dag_etl_paises
AIRFLOW_CTX_TASK_ID=crear_tabla_
AIRFLOW_CTX_EXECUTION_DATE=2023-07-09T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-07-09T00:00:00+00:00
[2023-07-28 22:13:35,734] {base.py:68} INFO - Using connection ID 'redshift_belen' for task execution.
[2023-07-28 22:13:37,559] {dbapi.py:231} INFO - Running statement: 
CREATE TABLE IF NOT EXISTS bapintor_coderhouse.ciudades (
    city VARCHAR PRIMARY KEY,
    "Housing" DECIMAL(10, 2),
    "Cost of Living" DECIMAL(10, 2),
    "Startups" DECIMAL(10, 2),
    "Venture Capital" DECIMAL(10, 2),
    "Travel Connectivity" DECIMAL(10, 2),
    "Commute" DECIMAL(10, 2),
    "Business Freedom" DECIMAL(10, 2),
    "Safety" DECIMAL(10, 2),
    "Healthcare" DECIMAL(10, 2),
    "Education" DECIMAL(10, 2),
    "Environmental Quality" DECIMAL(10, 2),
    "Economy" DECIMAL(10, 2),
    "Taxation" DECIMAL(10, 2),
    "Internet Access" DECIMAL(10, 2),
    "Leisure & Culture" DECIMAL(10, 2),
    "Tolerance" DECIMAL(10, 2),
    "Outdoors" DECIMAL(10, 2),
    "process_date" DATETIME DISTKEY
);, parameters: None
[2023-07-28 22:13:37,875] {postgres.py:94} INFO - INFO:  Relation "ciudades" already exists and will be skipped

[2023-07-28 22:13:39,666] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=dag_etl_paises, task_id=crear_tabla_, execution_date=20230709T000000, start_date=20230728T221324, end_date=20230728T221339
[2023-07-28 22:13:40,142] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-07-28 22:13:41,767] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
